component/_buttons.scss

```diff
@@ -1 +1,57 @@
-// @file buttons.scss
\ No newline at end of file
+// @file buttons.scss
+
+$form-button-background: palette(brand);
+$form-button-border: false;
+$form-button-border-color: palette(brand, dark);
+$form-button-border-width: 1px;
+$form-button-border-radius: $default-border-radius;
+$form-button-padding: .325em 1em .25em;
+
+// Default button style. Classes must be chained to change colors, i.e. class="button button--primary"
+a.button,
+button.button,
+input.button {
+	border-radius: $form-button-border-radius;
+	cursor: pointer;
+	margin-bottom: .5em;
+	padding: $form-button-padding;
+	@include appearance(button);
+	@include transition( all 250ms ease );
+}
+
+// Additional button styles must be chained, i.e. class="button button--primary"
+.button {
+	background-color: palette(mono, 25);
+	@if $form-button-border {
+		border: $form-button-border-width solid $form-button-border-color;
+	} @else {
+		border: none;
+	}
+	color: palette(mono, 66);
+	text-align: center;
+	
+	@include touch-hover() {
+		background-color: palette(mono, 10);
+		color: palette(mono, 75);
+	}
+
+	&--primary {
```

Comment:
I have a button mixin also to add into scaffold. I would try as much as possible to stay away from using predefined classes as for the most part we may not have access to code/classes without some programmatic implementations.

My mixin basically take a parameter such as `$primary` that could be a pplied to any buttons via css.